tests/lib/rules/no-unsupported-features/node-builtins.js

```diff
@@ -5305,6 +5305,12 @@ new RuleTester({ languageOptions: { sourceType: "module" } }).run(
                         node: { version: "12.11.0" },
                     },
                 },
+
+                // https://github.com/eslint-community/eslint-plugin-n/issues/253
+                {
+                    code: `fetch("/api/xxx");`,
+                    settings: { node: { version: ">=18.2.0" } },
```

Comment:
Does this not need to be `>=18.0.0` to match this?

https://github.com/eslint-community/eslint-plugin-n/blob/a0b45ee9c35279c29d3602609dad3b36d95d6f5f/lib/rules/no-unsupported-features/es-builtins.js#L604-L606

Reply:
well, I think it's worthy adding a few more tests.

Reply:
Found another problem: `fetch` should belong to node-builtins, not es-builtins? 🤔

Reply:
It should probably belong to both, as the es-builtins are all generated 👀 https://github.com/mdn/browser-compat-data + the script here https://github.com/eslint-community/eslint-plugin-n/pull/174#issuecomment-1913255700

Reply:
Technically it is a web api (whatwg). see https://github.com/mdn/browser-compat-data/blob/712e11aa4181a6c92aa552690c93e0354038b705/api/_globals/fetch.json#L3-L6

Reply:
I do think it's another issue (#260), and I just reverted the commit. The pr is now aimed to fix #250.